### DISCUSSION TITLE
Navigate to position

### DISCUSSION THREAD

**Discussion by ghost** [mrbungle64/ecovacs-deebot.js#316] [Ideas]
Would it be possible to give the deebot the command to move to a certain (preset) position? I know there is a getpossition command, would be great is it was accompanied bij a setposition command. The deeboot should just sit idle at that possition until it it get a command to return to the dock or start a cleaning sequence.

Thanks

**Comment by mrbungle64** [comment]
@cactusss 

For T8 and similar models:
```js
const goToPositionValues = x + ',' + y;
const area = goToPositionValues + ',' + goToPositionValues;
vacbot.run('CustomArea_V2', area, 1, 1);
```

For T9, T10 and X1 and similar models:
```js
const goToPositionValues = x + ',' + y;
const area = goToPositionValues;
vacbot.run('MapPoint_V2', area);
```

**Reply by ghost** [reply]
Nice! I'm gonna fiddle with it. Thanks!

**Reply by mrbungle64** [reply]
@cactusss 

Starting with version 0.9.5-alpha.0 you can also use the `GoToPosition` shortcut function, so that you don't have to handle the 2 variants anymore.

```js
const goToPositionValues = x + ',' + y;
vacbot.run('GoToPosition', goToPositionValues);
```
